charts/streams-app/templates/deployment.yaml

```diff
@@ -103,6 +103,13 @@ spec:
             - name: "{{ .Values.configurationEnvPrefix }}_ERROR_TOPIC"
               value: {{ .Values.streams.errorTopic | quote }}
           {{- end }}
+          {{- range $key, $value := .Values.secrets }}
+            - name: "{{ $key }}"
+              valueFrom:
+                secretKeyRef:
+                  name: "{{ $key | lower}}"
+                  key: value
```

Comment:
What if I have multiple secrets? Are there conflicts if it is always "value"?

Reply:
No, I don't think so, but I didn't test it. You could also use it for saving `username` and `password` in the same secret. See here: https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-environment-variables

Should I test it shortly?

Reply:
Sounds fine but testing is never a bad idea :)

Reply:
```
ABC:  <set to the key 'value' in secret 'abc'>  Optional: false
FOO:  <set to the key 'value' in secret 'foo'>  Optional: false
```

Looks fine to me in the description of the pod.